buildViaDocker.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Build the image
+eval "echo \"Building the image\""
+eval "docker build . -t protobufbuild"
+eval "echo \"Finished building the image\""
+
+# Terminate any existing containers.
+eval "echo \"Stopping previously ran containers and removing them\""
+eval "docker container stop protobufBuild && docker container rm protobufBuild"
+eval "echo \"Finished deleting previously ran containers.\""
+
+# Run the image.
+eval "echo \"Running the image.\""
+eval "docker run --name protobufBuild protobufbuild"
+eval "echo \"Completed running.\""
+
+# go to the correct folder.
+eval "echo \"Changing to src folder\""
+eval "cd src"
+eval "echo \"Completed changing to src folder.\""
+
+# copy back out
+eval "echo \"Copying libprotobuf-lite back out of the container.\""
```

Comment:
The location of the lib isn't `echo`d.  I had to check the script to find the lib.

Reply:
I'll add it in